configs/sst_platform_tools-gcc-toolset.yaml

```diff
@@ -34,5 +34,24 @@ data:
   - dwz
   - chrpath
 
+  package_placeholders:
+    gcc-libraries: This package might be needed to support applications built with the GCC Toolset.
+    requires:
```

Comment:
There needs to be one more indentation from here on.

The structure is:

```
  package_placeholders:
    package-name:
      description: This package is not in Fedora (yet), but we want to see it here.
      requires:
      - pkg1
      - pkg2
      buildrequires:
      - pkg2
      - pkg3
```

Reply:
Oop, fixed in the following commit.